### DISCUSSION TITLE
Using AWS IAM to authenticate

### DISCUSSION THREAD

**Discussion by ivanbishop** [emqx/emqx#7626] [Q&A]
Hi, does anyone have experience they can share on using IAM with EMQx (current version)?
Thanks
Ivan

NB.
Just saw a mention here, it may not be applicable.
 
https://www.bevywise.com/mqtt-broker/mqtt-security.html

**Comment by zmstone** [comment]
Hi @ivanbishop 

Do you mean for the authentication of EMQX dashboard user or MQTT clients ?
